src/main/java/com/uber/cadence/migration/MigrationInterceptor.java

```diff
@@ -70,8 +71,10 @@ public byte[] executeWorkflow(
         // Skip migration on non-cron and child workflows
         WorkflowExecutionStartedEventAttributes startedEventAttributes =
             input.getWorkflowExecutionStartedEventAttributes();
-        if (startedEventAttributes.cronSchedule == ""
-            || startedEventAttributes.getParentWorkflowExecution().getWorkflowId() != "") {
+        if (startedEventAttributes.cronSchedule == null
```

Comment:
nit: use isSet method
```suggestion
        if (!startedEventAttributes.isSetCronSchedule()
```